jwql/shared_tasks/run_pipeline.py

```diff
@@ -244,6 +317,7 @@ def run_save_jump(input_file, short_name, work_directory, instrument, ramp_fit=T
     parser.add_argument('input_file', metavar='FILE', type=str, help=file_help)
     parser.add_argument('short_name', metavar='NAME', type=str, help=name_help)
     parser.add_argument('max_cores', metavar='CORES', type=str, help=cores_help)
+    parser.add_argument('--step_args', metavar='STEP_ARGS', type=json.loads, default='{}', help=step_args_help)
```

Comment:
In the shared tasks log, it looks like there's a consistent warning message:

```
07/13/2023 15:45:59 PM WARNING: step_args: {'dark_current': {'skip': True}}
07/13/2023 15:45:59 PM WARNING: step_args_str: '{"dark_current":{"skip":"True"}}'
07/13/2023 15:45:59 PM WARNING: expected: {"dark_current": {"skip";"True"}}
```

So, based on the above, I'm not sure if the arguments are getting through un-mangled. Which in turn raises the questions:

- Is skipping the dark step actually working right now?
- If it is, is it working 'by accident' (e.g. because the string 'true' evaluates to the bool True, but the string 'false' would also evaluate to True, so we're not actually sending a boolean here)

Reply:
Hmm, wait. I do see them. I'll look into it.

Reply:
After restarting celery, these logging entries are now gone, as expected.